command/init.go

```diff
@@ -29,9 +32,18 @@ func (c *InitCommand) Run(args []string) int {
 		return 1
 	}
 
+	if shares == 0 {
+		if pgpKeys == nil {
+			shares = 5
+		} else {
+			shares = len(pgpKeys)
```

Comment:
Lets not do this, I can see people fucking this up accidentally. Lets just enforce len(keys) == shares.